assets/src/landing/stores/landing-store.tsx

```diff
@@ -1,47 +1,39 @@
-import { fromPromise, IPromiseBasedObservable } from 'mobx-utils';
 import { action, makeObservable, observable } from 'mobx';
-import { Notificator } from './notificator';
-import { TokenStorage } from './token-storage';
-import { getSupervisors } from '~/api/use-get-supervisors';
-import { manageSupervisor } from '~/api/use-manage-supervisors';
+import { useInvalidateSupervisors } from '~/api/use-get-supervisors';
+import { toastManager } from '~/shared/lib/toastManager';
+import { manageTokenStorage } from '~/shared/lib/token-storage';
 
 export class LandingStore {
```

Comment:
Not bad. Let's try to get rid of this class completely, I think it's unnecessary

Reply:
All right, I'm working on it. 😉️

Reply:
How about I dribble the current changes for now and you remove it permanently in the next PR?

Reply:
Sounds like a good idea! I'll do the next PR